python/lsst/ap/verify/ingestion.py

```diff
@@ -100,7 +100,7 @@ class DatasetIngestConfig(pexConfig.Config):
         doc="Path to top level of the defect tree.  This is a directory with a directory per sensor."
     )
     defectIngester = pexConfig.ConfigurableField(
-        target=IngestDefectsTask,
+        target=IngestStdTextCalibsTask,
```

Comment:
Is `IngestStdTextCalibsTask` a drop-in replacement for `IngestDefectsTask`? If there are any API changes, our current tests won't pick them up, because this code is no longer used for DECam data.

Reply:
Looking at lsst/obs_decam#123, the command-line interface is the same, and that's unfortunately what `DatasetIngestTask` has to use.

Reply:
Are there more changes required?  It was meant as a drop-in replacement.

Reply:
No, there aren't.